src/lib/connectors/repos/queries/gauging-stations.js

```diff
@@ -33,6 +34,7 @@ and lgs.date_deleted is null
 and gs.gauging_station_id=:gaugingStationId
 union all
 select
+lgs.licence_gauging_station_id,
```

Comment:
Please note you can also get this from `gs.gauging_station_id` - same outcome but slightly less computational power.

Reply:
Ignore this - I was mistaken.